### PR TITLE
Add theme of dark 

### PR DESCRIPTION
**Pull request by jwangyangls**
Signed-off-by: Yogi_Wang <yawang@vmware.com>
env:https://10.185.26.151
1.Bread crumbs change
2. add theme of dark

### CODE REVIEW

File: src/portal/src/app/theme.service.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        themeLink.href = styleName;
+    } else {
+        const style = this.document.createElement('link');
+        style.id = 'client-theme';
+        style.rel = 'stylesheet';
+        style.href = `${styleName}`;
+
+        head.appendChild(style);
+    }
+    // tslint:disable-next-line: no-unused-expression
```
Comment (AllForNothing) [inline src/portal/src/app/theme.service.ts]: unused code?

File: src/portal/src/app/base/navigator/navigator.component.html [hunk 2]
```diff
@@ -44,5 +44,16 @@
...
+        <ng-container *ngFor="let theme of themeArray;let i=index">
+                <ng-container *ngIf="theme.showStyle === styleMode">
+                    <a class="nav-link nav-icon theme-select"
+                        (click)="themeChanged(theme)">
+                        <clr-icon size="20" *ngIf="styleMode ==='DARK'" shape="sun" class="is-solid"></clr-icon>
+                        <clr-icon size="20" *ngIf="styleMode ==='LIGHT'" shape="moon" class="is-solid"></clr-icon>
+                        {{ theme.text | translate }}
+                    </a>
+                </ng-container>
+            </ng-container>
```
Comment (zhoumeina) [inline src/portal/src/app/base/navigator/navigator.component.html]: Please format this session of code to make sure the beginning of tag is in has same indent as closing one.

File: src/portal/src/css/dark-theme.scss [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+.harbor-icon {
+  filter: drop-shadow(#ccc 58px 2px) !important;
+}
+
+.color-white-dark {
+  color: white;
+}
+
+.labelFilterPanel {
```
Comment (zhoumeina) [inline src/portal/src/css/dark-theme.scss]: Why here using cameral case? not dash?

File: src/portal/src/styles.css [hunk 4]
```diff
@@ -3,7 +3,7 @@ body {
 }
 
 .app-loading {
-    position: absolute;
+    position: absolute ;
```
Comment (zhoumeina) [inline src/portal/src/styles.css]: why here is a blank before ;

File: src/portal/src/css/dark-theme.scss (hunk 3 above)
Comment (jwangyangls) [reply src/portal/src/css/dark-theme.scss]: This is a historical reason, I have modified

**Pull request merged**